### PR TITLE
Add section to Guide_CLIMADA_Tutorial.ipynb 

### PR DESCRIPTION
**Pull request by NicolasColombi**
Changes proposed in this PR:

 - Add a section in the guide doc/guide/Guide_CLIMADA_Tutorial.ipynb that specify to only use markdown instead of html to create tutorials.
 - Add an image of 46.7 KB

### PR Author Checklist

- [x] Read the [Contribution Guide][contrib]
- [x] Correct target branch selected (if unsure, select `develop`)
- [x] Descriptive pull request title added
- [x] Source branch up-to-date with target branch
- [x] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#2.--Commenting-&-Documenting) updated
- [ ] [Tests][testing] updated
- [x] [Tests][testing] passing
- [x] No new [linter issues][linter]
- [ ] [Changelog](https://github.com/CLIMADA-project/climada_python/blob/main/CHANGELOG.md) updated

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Reviewer_Checklist.html) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/main/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html
[linter]: https://climada-python.readthedocs.io/en/stable/guide/Guide_Continuous_Integration_and_Testing.html#3.C.--Static-Code-Analysis

### CODE REVIEW

File: doc/guide/Guide_CLIMADA_Tutorial.ipynb [hunk 1]
```diff
@@ -99,8 +102,86 @@
...
+    "\n",
+    "**'#'** : create a header of level 1\n",
+    "\n",
+    "**'##'** : create a header of level 2 \n",
+    "\n",
+    "**'###'** : create a header of level 3 \n",
+    "\n",
+    "**'####'** : create a header of level 4 \n",
+    "\n",
+    "The title of the tutorial should be of level 1 (#), should have his own cell and should be the first cell of the notebook. "
```
Comment (chahank) [inline doc/guide/Guide_CLIMADA_Tutorial.ipynb:138]:
```suggestion
    "The title of the tutorial should be of level 1 (#), should have its own cell, and should be the first cell of the notebook."
```

File: doc/guide/Guide_CLIMADA_Tutorial.ipynb [hunk 2]
```diff
@@ -85,8 +88,8 @@
     "  - keep the size of data you use as examples in the tutorial in mind\n",
     "  - we aim for computational efficiency\n",
     "  - a lean, well-organized, concise notebook is more informative than a long, messy all-encompassing one.\n",
-    "\n",
     "- follow the general CLIMADA naming convention for the notebook. For example: \"climada_hazard_TropCyclone.ipynb\"\n",
+    "\n",
     "![image-2.png](attachment:image-2.png)"
```
Comment (chahank) [inline doc/guide/Guide_CLIMADA_Tutorial.ipynb:93]: It is not part of this pull request, but why do we have this image here?

File: doc/guide/Guide_CLIMADA_Tutorial.ipynb [hunk 3]
```diff
@@ -99,8 +102,86 @@
...
+    "-\n",
+    "\n",
+    "\n",
+    "Instead of numbers :\n",
+    "\n",
+    "1.\n",
+    "2.\n",
+    "\n",
+    "\n",
+    "Lastly, the table of content should only contain headers of level 2 (##). Remember that links to headers are case sensitive!\n",
```
Comment (chahank) [inline doc/guide/Guide_CLIMADA_Tutorial.ipynb:163]: We should probably here indicate what "links to headers" are? How does one know what is the correct link for a given header?

**Comment by chahank** [comment]
Thanks, very useful!

File: doc/guide/Guide_CLIMADA_Tutorial.ipynb (hunk 3 above)
Comment (NicolasColombi) [reply doc/guide/Guide_CLIMADA_Tutorial.ipynb]: I added a few sentences to explain it, it should be more explicit now

File: doc/guide/Guide_CLIMADA_Tutorial.ipynb (hunk 2 above)
Comment (NicolasColombi) [reply doc/guide/Guide_CLIMADA_Tutorial.ipynb:93]: Good question, I do not really know.. shall I remove it ? I added a space just because it was attached to the text.

Comment (chahank) [reply doc/guide/Guide_CLIMADA_Tutorial.ipynb:93]: Yes, let us remove it.

File: doc/guide/Guide_CLIMADA_Tutorial.ipynb (hunk 3 above)
Comment (chahank) [reply doc/guide/Guide_CLIMADA_Tutorial.ipynb]: perfect.

File: doc/guide/Guide_CLIMADA_Tutorial.ipynb [hunk 4]
```diff
@@ -99,8 +100,90 @@
...
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Input:\n",
+    "\n",
+    "![](img/Table_content.png)\n",
```
Comment (emanuel-schmid) [inline doc/guide/Guide_CLIMADA_Tutorial.ipynb:177]: Nice picture - but wouldn't it be more useful to have it as plain text instead?

Comment (NicolasColombi) [reply doc/guide/Guide_CLIMADA_Tutorial.ipynb:177]: @emanuel-schmid  I definitely  agree it would be better just text. I just didn't know how to write it without rendering the output. Since it is Markdown, it is interpreted as such and therefore converted to a table. Do you know how to write Markdown syntax without rendering the output ?

**Comment by chahank** [comment]
@emanuel-schmid : I think this is ready to merge right?

File: doc/guide/Guide_CLIMADA_Tutorial.ipynb (hunk 4 above)
Comment (emanuel-schmid) [reply doc/guide/Guide_CLIMADA_Tutorial.ipynb]: @NicolasColombi : yes 😉 have a look at the changed guide
